### ISSUE TITLE
[FEATURE REQUEST] Indicate config apply failure in UI

### ISSUE THREAD

**Issue by ValeriiVozniuk** [WillyXJ/facileManager#604]
fM Version : **4.5.0**
fmDNS Version : **5.3.3**

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [x] I have read and understood the [contributors guide](https://github.com/WillyXJ/facileManager/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [x] I have checked that the issue I'm posting isn't already reported.
- [x] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/WillyXJ/facileManager/issues?q=is%3Aissue+is%3Aclosed) and [opened issues](https://github.com/WillyXJ/facileManager/issues)
- [x] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

---

**(BUG | ISSUE) Expected Behavior:**
In case, if there was a Build Config failure on one of the servers, indicate this in UI until next successful apply

**(BUG | ISSUE) Actual Behavior:**
```
nameserver01.example.com
 --> [nameserver01.example.com] Configuration build was successful.

nameserver02.example.com
 --> Failed: could not access nameserver02.example.com (tcp/22).

nameserver03.example.com
 --> [nameserver03.example.com] Configuration build was successful.

Complete.
```
Despite second server apply has failed, there are no indications about it in UI afterwards.

**(BUG | ISSUE) Steps to reproduce:**
1. Create a 3 node deployment.
2. Define some test zone and apply it to servers.
3. Shutdown the second server.
4. Make a change to created zone records, and run Build Config.
5. Apply on the second server will fail, but after clicking OK it is not seen anywhere. Even in Admin logs there are 2 records about successful build, but no error for the unsuccessful apply.

This scenario appeared yesterday during demo to the manager, and he was worried that support person may miss the fail during apply, and anyone would not notice that something is wrong afterwards.

**Comment by JonLaBass** [comment]
If a configuration build failed on a server, the servers page would still flag that server as needing it's configuration built along with the "process all updates" in the top bar. However, I will see about adding some graphical highlighting to the output dialog box to raise more attention.

**Comment by WillyXJ** [comment]
If a configuration build failed on a server, the servers page would still flag that server as needing it's configuration built along with the "process all updates" in the top bar. However, I will see about adding some graphical highlighting to the output dialog box to raise more attention.

**Comment by WillyXJ** [comment]
Will these sample styles be more noticeable?

![image](https://github.com/user-attachments/assets/4590323d-0894-48ae-9fdf-98fb2ac6bf56)

![image](https://github.com/user-attachments/assets/c8c07442-bd5e-4c9b-9458-34a2e2617505)

**Comment by ValeriiVozniuk** [comment]
I think yes, such color coding would definitely help on apply phase :)

**Comment by WillyXJ** [comment]
This is now implemented in [fM 4.7.0](http://www.facilemanager.com/download/) and later.
